ci/tests/test-financial-templates-lib.sh

```diff
@@ -8,7 +8,7 @@ cat << EOF
         command: ganache-cli -i 1234 -l 9000000 -p 9545
     working_directory: ~/protocol
     resource_class: medium+
-    parallelism: 35
+    parallelism: 10
```

Comment:
why change this?

Reply:
The parallelism was so high that we had _empty_ jobs that were failing because they were empty. I talked to @evaldofelipe  and he recommended that we reduce the parallelism to solve it. I also realized that each job was taking like < 2 mins (and most were like < 30s) with a pretty high variance. So I decided to reduce parallelism so ci would bunch some together without making the overall tests take longer (since startup time is the biggest cost for many of these). The results were pretty good so I kept it. We could probably reduce to even like 5 or so and we'd probably _still_ end up with the same long-pole time of like 2 mins, but 10 is probably good for now. Feel free to take a look at the job!